cypress/plugins/index.js

```diff
@@ -0,0 +1,311 @@
+      const warning = await cennznet.testAmountWarning(tab);
+      return warning;
+    },
+    withdrawToken: async ({ tokenSymbol, amount }) => {
+      const withdraw = await cennznet.withdrawToken({ tokenSymbol, amount });
+      return withdraw;
+    },
+    CENNZnetSignWithdrawal: async () => {
+      const signed = await cennznet.signWithdrawal();
+      return signed;
+    },
+    CENNZnetSwitchNetwork: async (networkString) => {
+      const switched = await cennznet.switchNetwork(networkString);
+      return switched;
+    },
+    // puppeteer commands
+    initPuppeteer: async () => {
+      const connected = await puppeteer.init();
+      return connected;
+    },
+    clearPuppeteer: async () => {
+      const cleared = await puppeteer.clear();
+      return cleared;
+    },
+    assignWindows: async () => {
+      const assigned = await puppeteer.assignWindows();
+      return assigned;
+    },
+    clearWindows: async () => {
+      const cleared = await puppeteer.clearWindows();
+      return cleared;
+    },
+    assignActiveTabName: async (tabName) => {
+      const assigned = await puppeteer.assignActiveTabName(tabName);
+      return assigned;
+    },
+    isMetamaskWindowActive: async () => {
+      const isMetamaskActive = await puppeteer.isMetamaskWindowActive();
+      return isMetamaskActive;
+    },
+    isCypressWindowActive: async () => {
+      const isCypressActive = await puppeteer.isCypressWindowActive();
+      return isCypressActive;
+    },
+    switchToCENNZnetWindow: async () => {
+      const switched = await puppeteer.switchToCENNZnetWindow();
+      return switched;
+    },
+    switchToCypressWindow: async () => {
+      const switched = await puppeteer.switchToCypressWindow();
+      return switched;
+    },
+    switchToMetamaskWindow: async () => {
+      const switched = await puppeteer.switchToMetamaskWindow();
+      return switched;
+    },
+    switchToMetamaskNotification: async () => {
+      const notificationPage = await puppeteer.switchToMetamaskNotification();
+      return notificationPage;
+    },
+    unlockMetamask: async (password) => {
+      const unlocked = await metamask.unlock(password);
+      return unlocked;
+    },
+    importMetamaskAccount: async (privateKey) => {
+      const imported = await metamask.importAccount(privateKey);
+      return imported;
+    },
+    createMetamaskAccount: async (accountName) => {
+      const created = await metamask.createAccount(accountName);
+      return created;
+    },
+    switchMetamaskAccount: async (accountNameOrAccountNumber) => {
+      const switched = await metamask.switchAccount(accountNameOrAccountNumber);
+      return switched;
+    },
+    addMetamaskNetwork: async (network) => {
+      const networkAdded = await metamask.addNetwork(network);
+      return networkAdded;
+    },
+    changeMetamaskNetwork: async (network) => {
+      if (process.env.NETWORK_NAME && !network) {
```

Comment:
do we need to include env variables in the CI as well? or is Kovan network all good for default testing network?

Reply:
no this is optional - Kovan is good for default and can call this method during testing to test network switching

Reply:
nice sounds good